data/active_adapters.py

```diff
@@ -116,6 +117,7 @@
     'mouse_variant': MouseGenomesProjectAdapter(filepath='./samples/mouse_variants/mouse_variant_snps_rsid_sample.vcf'),
     'variant_disease': ClinGen('./samples/clinGen_variant_pathogenicity_example.csv', label='variant_disease'),
     'variant_disease_gene': ClinGen('./samples/clinGen_variant_pathogenicity_example.csv', label='variant_disease_gene'),
+    'bluestarr_variant_elements': BlueSTARRVariantElement('./samples/variant_phenotype_V2F.example.tsv', label='variant_elements'),
```

Comment:
the label here needs to be consistent with what you are using in the adapter script (line 14), so should be variant_genomic_elements

Reply:
and seems we are following singular form convention for label, variant_genomic_elements -> could be variant_genomic_element

Reply:
Done.